pkg/plugin/cmd.go

```diff
@@ -787,6 +787,7 @@ func (o *DebugOptions) deleteAgent(agentPod *corev1.Pod) {
 	}
 	err := o.CoreClient.Pods(agentPod.Namespace).Delete(agentPod.Name, v1.NewDeleteOptions(0))
 	if err != nil {
-		fmt.Fprintf(o.ErrOut, "failed to delete agent pod[Name:%s, Namespace: %s], consider manual deletion.\n", agentPod.Name, agentPod.Namespace)
+		fmt.Printf("error in delete agent pod: %v\n", err)
```

Comment:
Use `fmt.Fprintf` so that the IO stream set by caller can get respected

Reply:
i find this bug.
https://github.com/aylei/kubectl-debug/blob/f08222faf14c18b975ccb5e0fc95152e9e512bcd/pkg/plugin/cmd.go#L491-L519
in `t.Safe(withCleanUp)`, all `fmt.Fprintf(o.ErrOut,.....)` will get panic error.
such as 
https://github.com/aylei/kubectl-debug/blob/f08222faf14c18b975ccb5e0fc95152e9e512bcd/pkg/plugin/cmd.go#L498
the error msg will never print.

the `o.ErrOut` is nil pointer. 
do you know why?
i'l trying to fixed it later.

Reply:
https://github.com/aylei/kubectl-debug/blob/f08222faf14c18b975ccb5e0fc95152e9e512bcd/pkg/plugin/cmd.go#L403-L409
after line408 `o.ErrOut = nil` the `fmt.Fprintf(o.ErrOut, .....)` will get panic.
maybe change the o.ErrOut to o.Out after line408 `o.ErrOut = nil`?